packages/muelu/test/maxwell/Maxwell3D.cpp

```diff
@@ -527,8 +524,8 @@ int MainWrappers<double,LocalOrdinal,GlobalOrdinal,Node>::main_(Teuchos::Command
       Teuchos::RCP<Thyra::LinearOpWithSolveBase<Scalar> > thyraInverseA = Thyra::linearOpWithSolve(*solverFactory, thyraA);
 
       comm->barrier();
-      tm = Teuchos::null;
-      tm = rcp(new TimeMonitor(*TimeMonitor::getNewTimer("Maxwell: 3 - Solve")));
+
```

Comment:
Hate demands you add a tm4= Teuchos::null here.

Reply:
Just push the PR and fix it later though.

Reply:
I just caught that one, and it's been pushed.

I'm waiting on serial builds to come back clean...